### PR TITLE
Fix Bug#284 -Resource added for Mocha unit test 

### PR DESCRIPTION
**Pull request by munishm**
Added link for unit testing azure function with Mocha JS

Bug@ 284

### CODE REVIEW

**Comment by munishm** [comment]
The official documentation on unit testing Javascript Azure functions is on Jest so the documentation& sample on how to unit test using Mocha JS is published it on Azure-Samples & put a reference here.

**Comment by munishm** [comment]
@fnocera - Can you please let me know what are the next steps?

**Comment by fnocera** [comment]
@munishm you should get two reviews and then go ahead and merge in your changes.

**Comment by omusavi** [comment]
Hey @munishm, thanks for helping out and contributing! We are currently planning on rewriting this page as well as changing up how we do technical recommendations. At least at first, we will be removing this section from the wiki page while we evaluate all the different options and how we want to present them. Given all that I do not think we need this PR to be merged, though I encourage you to come take a look once we have this all updated and feel free to let me know if you see anything that needs any additional information!

File: test-first-development/unit-testing/readme.md [hunk 1]
```diff
@@ -63,5 +63,6 @@ Languages and Platforms provide their own unit test tools and frameworks. In [CS
 
 - .NET Tutorial: <https://docs.microsoft.com/en-us/visualstudio/test/walkthrough-creating-and-running-unit-tests-for-managed-code>
 - Java Tutorial: <http://tutorials.jenkov.com/java-unit-testing/index.html>
+- Node Js Tutorial: <https://github.com/Azure-Samples/azure-functions-unit-testing-mocha>
```
Comment (davidkpiano) [inline test-first-development/unit-testing/readme.md:66]:
```suggestion
- Node.js Tutorial: <https://github.com/Azure-Samples/azure-functions-unit-testing-mocha>
```

**Comment by fnocera** [comment]
@omusavi given the plan do you want to go ahead and close the PR?

**Comment by fnocera** [comment]
@omusavi when will the updates for this file be completed? It's a small change, do you not think we should update the link in the mean time? If not then we should close the PR.

**Comment by fnocera** [comment]
Closing after conversation with @RichCzyzewski given @omusavi is working on a unit test re-write

**Pull request closed without merging**